File: deaf_grandma.js
function deafGrandma(str) {
let stringIsUppercase = true;
for (i = 0; i < str.length; i++){
    if (str.match(/[a-z]/g)){
        stringIsUppercase = false;
        break;
    }

}

if (str.length == 0){
    console.log(">")
    console.log("WHAT!?");
}else if(stringIsUppercase == false){
    console.log("> " + str);
    console.log("SPEAK UP, KID!");
}else if (str === "GOODBYE!"){
    console.log("> " + str);
    console.log("LATER, SKATER!");
}else if(stringIsUppercase == true){
    console.log("> " + str);
    console.log("NO, NOT SINCE 1946!");
}



}

deafGrandma("grandma run away");

File: deaf_grandma.py
def deaf_grandma():

   
    print("HEY KID!")
    
    number_of_goodbyes = 0
    
    while number_of_goodbyes < 2:
        user_input = input("Enter Input: ")
        
        if user_input == "GOODBYE!" and number_of_goodbyes < 1:
            print("LEAVING SO SOON?")
            number_of_goodbyes = 1
        elif user_input == "GOODBYE!" and number_of_goodbyes == 1:
            print("LATER, SKATER!")
            number_of_goodbyes = 2
        elif len(user_input) == 0:
            print("WHAT?!")
            number_of_goodbyes = 0
        elif user_input.isupper():
            print("NO, NOT SINCE 1946")
            number_of_goodbyes = 0
        elif user_input.isupper() == False:
            print("SPEAK UP, KID!")
            number_of_goodbyes = 0





deaf_grandma()